Classes/Log/Backend/GraylogBackend.php

```diff
@@ -49,21 +51,22 @@ public function append($message, $severity = LOG_INFO, $additionalData = null, $
 
         $output = $severityLabel . ': ' . $message;
 
-        $messageContext = [];
-        !is_null($packageKey) ? $messageContext['packageKey'] = $packageKey : '';
-        !is_null($className) ? $messageContext['className'] = $className : '';
-        !is_null($methodName) ? $messageContext['methodName'] = $methodName : '';
-        !is_null($additionalData) ? $messageContext['additionalData'] = $additionalData : '';
-        !is_null($ipAddress) ? $messageContext['ipAddress'] = $ipAddress : '';
-        !is_null($severityLabel) ? $messageContext['severityLabel'] = $severityLabel : '';
-        
-        if (!$this->graylogService instanceof  GraylogService) {
-            $this->graylogService = new GraylogService();
-        }
-        
-        $this->graylogService->logMessage($output, $messageContext, $severity);
+        $messageContext = [
+            'packageKey' => !is_null($packageKey) ? $packageKey : '',
+            'className' => !is_null($className) ? $className : '',
+            'methodName' => !is_null($methodName) ? $methodName : '',
+            'additionalData' => !is_null($additionalData) ? $additionalData : '',
+            'ipAddress' => !is_null($ipAddress) ? $ipAddress : '',
+            'severityLabel' => !is_null($severityLabel) ? $severityLabel : '',
+        ];
+        $this->getGraylogService()->logMessage($output, $messageContext, $severity);
```

Comment:
The `instanceof` check introduced with c29d18040 will always return false (at first run) because it's a `DependencyProxy` => copied `getGraylogService()` from `GraylogExceptionHandler`